sentry/src/main/java/io/sentry/MainEventProcessor.java

```diff
@@ -72,6 +75,27 @@
     return event;
   }
 
+  private void setDebugMeta(final @NotNull SentryEvent event) {
+    if (options.getProguardUuid() != null) {
+      DebugMeta debugMeta = event.getDebugMeta();
+
+      if (debugMeta == null) {
+        debugMeta = new DebugMeta();
+        debugMeta.setImages(new ArrayList<>());
+      } else if (debugMeta.getImages() == null) {
+        debugMeta.setImages(new ArrayList<>());
+      }
+      List<DebugImage> images = debugMeta.getImages();
```

Comment:
could be its own `if` so we don't duplicate the `debugMeta.setImages(new ArrayList<>());` part.

eg
```
if (debugMeta == null) {
	debugMeta = new DebugMeta();
}
if (debugMeta.getImages() == null) {
	debugMeta.setImages(new ArrayList<>());
}
```